docs(hub): add agent hub guides (#80)
agntcy/docs: docs/pages/hub/hub.md

Clicking the three dots (**⁝**) at the end of any record in opens a drop-down list of actions you can perform on that organization.

* Click **Edit** to edit the organization.
* Click **Delete** to delete the organization.
* Click **Switch** to switch to the organization.

You can reload the listed items by clicking the **Reload** button (**⟳**).

#### Users

The users in a organization are listed under the **Users** tab.

You can invite other users to the organization by clicking the **+ Invite User** button.

> Note:
> You cannot invite other users to your personal organization created during signing up. To collaborate with others, create a new organization.

Clicking the three dots (**⁝**) at the end of any record in opens a drop-down list of actions you can perform on that organization.

* Click **Edit** to edit the user's role.
* Click **Delete** to delete the user.

You can reload the listed items by clicking the **Reload** button (**⟳**).

### Using the Hub through CLI

You can use the Agent Hub through the CLI.

> Use the `bin/dirctl hub` command to list the available commands.

Reviewer: we can remove `bin/` prefix and only rely on `dirctl`

Author: Shuold `bin/` be removed from all instances or just here?

Reviewer: we should rename instances of `bin/dirctl` to `dirctl`

#### Initiate the Repository

```
dirctl repo init
```

#### Logging in

Use the `bin/dirctl hub login` command to log in. The login page opens in your browser. Use your credentials to log in.

#### Listing Organizations

> Use the `bin/dirctl hub tenants` command to list the organizations you are a member of.

Reviewer: @muscariello should we switch to orgs here?

Reviewer: @keraron decided, we will switch this to

```suggestion
Use the `dirctl hub orgs` command to list the organizations you are a member of.
```

> #### Pushing and Pulling Agent Data Models

Reviewer: we should include here details about signing and verification from:

```suggestion
#### Signing Agent Data Models

It is mandatory to sign the agent data models before pushing to Hub. Unsigned models will be rejected by the API. 
To sign an agent data model using identity-based OIDC signing, you can run `dirctl sign ./agent.json`

#### Pushing and Pulling Agent Data Models
```

To pull the data model, use the `bin/dirctl hub pull <repository>:<version>` command. Alternatively, you can use `bin/dirctl hub pull <digest>` instead.

To push the data model, use the `bin/dirctl hub push <agent> ./agent.json` command.

> 

Reviewer: ```suggestion

#### Verifying Agent Data Model signature

The verification process allows validation of agent data model signature against a specific identity.
To verify that an agent model is properly signed, you can run `dirctl verify ./agent.json`.
If you wish to verify the signature against a specific identity, for example to check if an agent model originates from GitHub Agntcy users, you can run `dirctl verify ./agent.json --oidc-identity "(.*)@agntcy.com" --oidc-issuer "(.*)github.com(.*)"`
```
